plugins/inputs/mikrotik/mikrotik.go

```diff
@@ -0,0 +1,328 @@
+//go:generate ../../../tools/readme_config_includer/generator
+package mikrotik
+
+import (
+	_ "embed"
+	"encoding/json"
+	"errors"
+	"fmt"
+	"github.com/influxdata/telegraf/plugins/inputs"
+	"io"
+	"net/http"
+	"reflect"
+	"strings"
+	"sync"
+	"time"
+
+	common_tls "github.com/influxdata/telegraf/plugins/common/tls"
+
+	"github.com/influxdata/telegraf"
+	"github.com/influxdata/telegraf/config"
+)
+
+const metricName = "mikrotik"
+
+//go:embed sample.conf
+var sampleConfig string
+
+var durationParseTags []string
```

Comment:
1. My review comments are not meant as insult or anything, I've asked that question because I wonder why this is necessary.
2. Then either make `shallWeParseTime` a member of the plugin and store the data there. The issue with this being global is that if you do have multiple instances of this plugin they will all share the same `durationParseTags` and therefore might break each other!